spec/inferno/utils/verify_runnable_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../../lib/inferno/utils/verify_runnable'
+
+RSpec.describe Inferno::Utils::VerifyRunnable do
+  describe '#verify_runnable' do
+    let(:dummy_class) { Class.new { include Inferno::Utils::VerifyRunnable } }
+    let(:dummy) { dummy_class.new }
+    let(:suite) { BasicTestSuite::Suite }
+    let(:group) { BasicTestSuite::AbcGroup }
+    let(:good_inputs) do
+      [{ name: 'input1', value: 'baz' }, { name: 'input2', value: 'foo' }]
+    end
+    let(:bad_inputs) { [{ name: :input2, value: :foo }] }
+    let(:unrunnable) { BasicTestSuite::DefGroup.tests.first }
+
+    it 'is defined' do
+      expect(described_class.method_defined?(:verify_runnable)).to eq(true)
+    end
+
+    it 'allows legit runnables' do
```

Comment:
Let's not use "legit" as a technical term in our code. I feel like this title should also mention the inputs, since we have a test which says it "rejects bad inputs", but none of them mention good inputs.